docs/DE0008.md

```diff
@@ -0,0 +1,17 @@
+<!--
+M:System.Activator.CreateInstance(System.Type,System.Reflection.BindingFlags,System.Reflection.Binder,System.Object[],System.Globalization.CultureInfo,System.Object[])
+M:System.Activator.CreateInstance(System.Type,System.Object[],System.Object[])
+-->
+
+# DE0008: Activator.CreateInstace overload shouldn't be used
```

Comment:
Is deprecation really the right thing here? Deprecation should be a global property of an API, i.e. independent of how specific APIs are implemented by specific platforms. If the APIs work fine on .NET Framework but just don't work on .NET Core, then deprecating the API seems over the top because it would basically mean all APIs that throw `PlatformNotSupported` would be marked as deprecated. That doesn't seem right.

Reply:
Yeah, it doesn't feel exactly right as deprecation. Let's chat about it.